GitUpKit/Views/GIMapViewController+Operations.m

```diff
@@ -40,6 +41,10 @@ - (void)_promptForCommitMessage:(NSString*)message withTitle:(NSString*)title bu
   return [message stringByTrimmingCharactersInSet:[NSCharacterSet newlineCharacterSet]];
 }
 
+static inline GIAlertType _AlertTypeForDangerousRemoteOperations() {
+	return ([[NSUserDefaults standardUserDefaults] boolForKey:kUserDefaultsKey_AllowReturnKeyForDangerousRemoteOperations] ? kGIAlertType_Stop : kGIAlertType_Danger);
```

Comment:
Indenting looks wrong. Make sure to set Xcode indenting to 2 spaces in the project info.